hardware/SensorPi/utils.py

```diff
@@ -0,0 +1,5 @@
+from datetime import datetime, timezone
```

Comment:
There is a module in the hardware directory called Util, this can be moved to that module.

Reply:
Moved the implementation under Utils module.

Reply:
Do we still need this file?

Reply:
Oh I missed it. Will delete this